shared/bash_remediation_functions/disable_prelink.sh

```diff
@@ -0,0 +1,16 @@
+function disable_prelink {
+	# Disable prelinking and don't even check
+	# whether it is installed.
+	if grep -q ^PRELINKING /etc/sysconfig/prelink
+	then
+		sed -i 's/^PRELINKING[:blank:]*=[:blank:]*[:alpha:]*/PRELINKING=no/' /etc/sysconfig/prelink
+	else
+		printf '\n' >> /etc/sysconfig/prelink
```

Comment:
I wish this wasn't a separate line but I won't hold up the merge because of it...